source/declarators.tex

```diff
@@ -2706,7 +2706,7 @@
 \begin{codeblock}
   int f(bool b) {
     unsigned char c;
-    unsigned char d = c; // OK, \tcode{d} has an indeterminate value
+    unsigned char d = c; // OK: \tcode{d} has an indeterminate value
```

Comment:
This one should probably be left alone, since it's really saying "OK, but `d` has an indeterminate value"